src/System.Management.Automation/FormatAndOutput/DefaultFormatters/Certificate_format_ps1xml.cs

```diff
@@ -52,9 +52,11 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Security_Cryptog
                     .GroupByProperty("PSParentPath")
                     .AddHeader(width: 41)
                     .AddHeader()
+                    .AddHeader()
                     .StartRowDefinition()
                         .AddPropertyColumn("Thumbprint")
                         .AddPropertyColumn("Subject")
+                        .AddPropertyColumn("EnhancedKeyUsageList")
```

Comment:
Does this show a list with many items correctly?

Reply:
If the console is wide enough, it shows up as comma separated list:

```none
   PSParentPath: Microsoft.PowerShell.Security\Certificate::LocalMachine\my

Thumbprint                                Subject                                                  EnhancedKeyUsageList
----------                                -------                                                  --------------------
FBC22CDCE5C3883FE36674993ED003C8438C0A79  CN=testcert.steve                                        {Code Signing (1.3.6.1.5.5.7.3.3)}
372B5224BC0182C7493D4C7501EAD1B127890C89  CN=testcert.steve                                        {Code Signing (1.3.6.1.5.5.7.3.3)}
9552B870CF7AB1D0506BC7E3EA89BC8A279E0723  CN=testcert.steve                                        {Client Authentication (1.3.6.1.5.5.7.3.2), Server A...
```

Perhaps I should set a fixed width for Subject so that more of EnhancedKeyUsageList shows up...

Reply:
Is it possible to show just the usage names without the OID's? I can always dig in if I need a specific OID for whatever reason, but when looking at a multi-usage key list quickly I care more about the soft-names than OIDs. That would also decrease the width needs of the `EnhancedKeyUsageList` column.

Reply:
@markekraus already ahead of you on that :)